custom_components/versatile_thermostat/translations/el.json

```diff
@@ -50,6 +51,7 @@
           "heater_entity2_id": "Προαιρετική 2η ταυτότητα οντότητας θερμαντήρα. Αφήστε κενό αν δεν χρησιμοποιείται",
           "heater_entity3_id": "Προαιρετική 3η ταυτότητα οντότητας θερμαντήρα. Αφήστε κενό αν δεν χρησιμοποιείται",
           "heater_entity4_id": "Προαιρετική 4η ταυτότητα οντότητας θερμαντήρα. Αφήστε κενό αν δεν χρησιμοποιείται",
+          "heater_keep_alive": "Optional heater switch state refresh interval. Disabled if empty or zero",
```

Comment:
I had a go at French and Italian because I had some hope of being able to make absurd Google translations slightly less absurd, but Greek and Slovak I will not even try. A copy-and-paste of Google Translate would end up as some nonsense like _“Option to replace the country’s freshen up intermission”_ (or worse).

So I entered English as placeholder for the Greek and Slovak translations. I suggest that we merge this PR (when you feel like it is tolerable), and straight after that we invite **`misa1515`, `vassilis-papanikolaou`** and **`Bergoglio`** to provide translations (either they create a new PR, or we do). We could also put this PR on hold until they provide translations, but this PR already has 40+ comments, touches 21 files, and if they took a while to reply, the PR could end up with merge conflicts.

Reply:
No. HA will do itself the fallback to the En version. Just don't add the values for language you don't know.

Some contributors make a PR regularly to adapt the text. I will ping them after the release so that they will be able to add their texte.